backend/experiment/views.py

```diff
@@ -175,20 +180,42 @@ def serialize_experiment_series(
     }
 
 
-def serialize_experiment(experiment_object: Experiment, finished=0):
+def serialize_experiment(experiment_object: Experiment, started=0, finished=0):
     return {
         'slug': experiment_object.slug,
         'name': experiment_object.name,
+        'started_session_count': started,
         'finished_session_count': finished,
         'description': experiment_object.description,
         'image': experiment_object.image.file.url if experiment_object.image else '',
     }
 
 
-def get_finished_session_count(experiment, participant):
-    ''' Get the number of finished sessions for this experiment and participant '''
+def get_started_session_count(
+        experiment: Experiment,
+        participant: Participant
+        ):
+    ''' Get the number of started sessions for
+        this experiment and participant '''
     count = Session.objects.filter(
-        experiment=experiment, participant=participant, finished_at__isnull=False).count()
+        experiment=experiment,
+        participant=participant,
+        started_at__isnull=False
```

Comment:
By definition, started_at is not null, since this value will always be filled once a session is created. So this line can be left out.